twitter_scrooge/twitter_scrooge.bzl

```diff
@@ -127,7 +127,8 @@ def _list_to_map(items):
 def _gen_scrooge_srcjar_impl(ctx):
   remote_jars = []
   for target in ctx.attr.remote_jars:
-    remote_jars.append(_jar_filetype.filter(target.files))
+    remote_jars.append(depset(_jar_filetype.filter(target.files)))
```

Comment:
Same thought about having a test for this

Reply:
Adding a test for this is pretty involved I believe since we need to have a remote jar built/setup, and its broken now so i'd rather not do this as part of this PR.

Reply:
Umm nevermind, turns out we had enough stuff in place the test was pretty easy to add.